spec/fixtures/actor.rb

```diff
@@ -30,6 +34,14 @@ class ActorSerializer < UserSerializer
   ) do |object|
     object.movies
   end
+
+  def symbol_conditionals_off(_object, params)
+    params[:symbol_conditionals_off].nil?
+  end
+
+  def show_birthplace(object)
+    object.show_birthplace
+  end
```

Comment:
I don't think we need both conditionals here. I think having just one (for ex. `age`) is more than enough since the interface of the conditional method will always provide the object and the params (based on `FastJsonapi#call_proc`) as the arguments. So it's up to the end-user to decide what to do with it.

Let me know if I'm not missing anything here...